### ISSUE TITLE
Node (10) Runner Reached End of Life

### ISSUE THREAD

**Issue by rbosai** [masamitsu-murase/publish_markdown_reports#11]
Hello,

Currently we are making use of your extension in our azure devops pipelines.
```
==============================================================================
Task         : Publish Markdown reports
Description  : A task to publish Markdown reports in a tab in your build result.
Version      : 1.7.0
Author       : Masamitsu MURASE
Help         : 
==============================================================================
```
During these builds the following warning is displayed:
`##[warning]Task 'Publish Markdown reports' version 1 (PublishMarkdownReports@1) is dependent on a Node version (10) that is end-of-life. Contact the extension owner for an updated version of the task. Task maintainers should review Node upgrade guidance: https://aka.ms/node-runner-guidance`

Which seems to be in line with the runner that is referenced in the [task.json](https://github.com/masamitsu-murase/publish_markdown_reports/blob/develop/PublishMarkdownReports/task.json#L50-L54) file.

Since the node version is end of life, are there any plans to upgrade this extension to make use of a more modern Node version that is not EOL?

If there is anything I overlooked that I can do on my side to resolve this warning then please let me know as well.

**Comment by masamitsu-murase** [comment]
Thank you very much for the report.
I released v1.8.1 to support Node 20.

**Comment by masamitsu-murase** [comment]
d5e8c9b06707d3e29d0d8af78e2609e8653a754c fixed this issue.

**Comment by rbosai** [comment]
> Thank you very much for the report. I released v1.8.1 to support Node 20.

Thank you very much as well for your work and your quick response to the issue!
